weblate/templates/snippets/project/billing-libre-trial.html

```diff
@@ -8,6 +8,6 @@
   {% if user_can_view_billing %}
     <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for libre hosting" %}</a>
   {% endif %}
-  {% trans "This project is in the trial period, so be cautious before you contribute. Things like license or repository can still change before being approved as a libre project." %}
+  {% trans "Save your work on this libre project till it is fully set up and accepted. Its libre license may change." %}
```

Comment:
↑ Save your work on this libre project till the libre license and everything else is fully set up and accepted.